packages/drivers/file-driver/api-report/file-driver.api.md

```diff
@@ -4,42 +4,46 @@
 
 ```ts
 
-import * as api from '@fluidframework/protocol-definitions';
```

Comment:
The `flub modify fluid-imports` tool does not support the '* as api' pattern.

I semi-automatically expanded these into normal imports so that I could use `flub modify fluid-imports` to sort '/' vs '/internal' and so that we can use `flub modify` to adjust imports in case we retag anything in the future.